GOVERNANCE.md

```diff
@@ -25,4 +26,25 @@ Contributors are encouraged to ask for collaborator access once you have had a f
 
 ## Process for releasing a new version of Wikimate
 
-[TODO]
+A PR should be created with all relevant changes to update the repository for the upcoming release.
+(See [#81](https://github.com/hamstar/Wikimate/pull/81) for an example.)
+It should apply the following actions:
+
+1. Change the "Unreleased" heading in the CHANGELOG.md file
```

Comment:
I see what you mean, but yeah, I guess people do end up getting used to the github model. I never even considered any other interpretation until you brought this up :)

Now that I've thought about it more explicitly, I do believe this is a better way of working with FOSS, as it promotes openness and transparency of the development process: unpolished changes go in separate branches (and WIP commits can be easily squashed, amended, etc); changes ready for review by maintainers go to the PR stage; changes considered ready are merged into master; and changes that have been on master for a while (hopefully tested by the most intrepid users who prefer not to wait for releases) are released with a stable (immutable) tag, downloadable as an archive, bundled with release notes, supported by the maintainers, etc.

Given this, I don't think we should word the top section based on the premise than master is a "release". If none of the options appeal to you, I agree, let's stick to what Keep a Changelog suggests.